### PR TITLE
Added documentation for out of space and certificate error

### PR DESCRIPTION
**Pull request by altendorfme**
Documentation for issues https://github.com/jamesog/tailscale-edgeos/issues/29 e https://github.com/jamesog/tailscale-edgeos/issues/15

### CODE REVIEW

**Comment by elitzer2** [comment]
Just ran into this issue and these directions resolved my issues. Thanks!

**Comment by altendorfme** [comment]
> Just ran into this issue and these directions resolved my issues. Thanks!

I need to be honest and confess that I migrated EdgeRouter-X to OpenWRT, there is already a ready-made package for TailScale and other VPNs, it was much easier than I imagined and the project continues to be updated, unlike EdgeOS with Unifi.

File: README.md [hunk 1]
```diff
@@ -125,6 +125,21 @@ Then copy the latest version:
 cp /var/cache/apt/archives/tailscale_*.deb /config/data/firstboot/install-packages
 ```
 
+If you still receive an **out of space** error when upgrading, try cleaning the system's images using:
+
+```
+delete system image
+```
+
+If you have a **certificate error** when upgrading, unfortunately it is a [Unifi problem](https://community.ui.com/questions/Fix-Solution-Lets-Encrypt-DST-Root-CA-X3-Expiration-Problems-with-IDS-IPS-Signature-Updates-HTTPS-E/0404a626-1a77-4d6c-9b4c-17ea3dea641d), but to correct it manually you can use
```
Comment (jamesog) [inline README.md:134]:
```suggestion
If you have a **certificate error** when upgrading, unfortunately it is an [EdgeOS problem](https://community.ui.com/questions/Fix-Solution-Lets-Encrypt-DST-Root-CA-X3-Expiration-Problems-with-IDS-IPS-Signature-Updates-HTTPS-E/0404a626-1a77-4d6c-9b4c-17ea3dea641d), but to correct it manually you can run the following commands:
```

**Review by jamesog — COMMENTED** [review]
Thanks very much for putting this together! I have a small edit I'd like before merging this.

**Pull request merged**